### PR TITLE
Prototype for "View Meeting" page

### PR DESCRIPTION
**Pull request by anna-giang**
Added a "View Meeting" page that users will use to view meeting details of the meetings they created or are invited to.
Users will also use this page to vote for and propose additional meeting times, the prototype of the form for this has been added here.

### CODE REVIEW

**Review by vivianho18 — COMMENTED** [review]
LGTM 👍

File: prototype/src/main/webapp/script.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+PROPOSED_TIMES_NO = 3; // Initially we hardcoded 3 different times suggestions
```
Comment (ghost) [inline prototype/src/main/webapp/script.js]: Use `let`, also keep this camelCase as it is not a constant.

File: prototype/src/main/webapp/script.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+function voteTime() {
+  // get the answers from the form and update votes on screen
+  // NOTE this data would be sent to a servlet in MVP and the data fetched to update UI
+  let inputs = document.getElementById("vote-time").elements;
+  for (let i = 0; i < PROPOSED_TIMES_NO; i++) {
+    if (inputs[i].checked === true) {
+      document.getElementById(inputs[i].id + "-vote").innerText = (
```
Comment (ghost) [inline prototype/src/main/webapp/script.js]: I would make `document.getElementById(inputs[i].id + "-vote")` into it's own variable as it makes it clearer that the code is incrementing the element's value by one.

File: prototype/src/main/webapp/view-meeting.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        <table id="times-table">
+          <tr>
+            <th>Option</th>
+            <th>Time</th>
+            <th>Current Votes</th>
+            <th>Vote</th>
+          </tr>
+          <tr>
+            <th>1</th>
+            <th>Thursday, 14th January 10:00</th>
```
Comment (ghost) [inline prototype/src/main/webapp/view-meeting.html]: `th` is used for headers of table rows/columns - you want `td` here (and all the other data cells in this table)

File: prototype/src/main/webapp/view-meeting.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Meeting</title>
+    <script src="script.js"></script>
```
Comment (ghost) [inline prototype/src/main/webapp/view-meeting.html]: I know this is a prototype, but surely there's a better name than `script.js`

Comment (anna-giang) [reply prototype/src/main/webapp/view-meeting.html]: Renamed it to viewMeeting.js 😅

File: prototype/src/main/webapp/script.js (hunk 1 above)
Comment (anna-giang) [reply prototype/src/main/webapp/script.js]: Yep, done!

File: prototype/src/main/webapp/script.js (hunk 2 above)
Comment (anna-giang) [reply prototype/src/main/webapp/script.js]: Okay, changed it!

File: prototype/src/main/webapp/view-meeting.html (hunk 3 above)
Comment (anna-giang) [reply prototype/src/main/webapp/view-meeting.html]: Oh right, thank you!

**Review by ghost — APPROVED** [review]
LGTM

Also as a general comment, for PRs that add/change html code, it's good practice to also attach a screenshot of the webpage. This helps reviewers visualise the new changes too.

**Pull request merged**